locale/pt_BR/manageiq.po

```diff
@@ -10514,9 +10517,15 @@ msgstr "ConfigurationScriptBase | Saída"
 msgid "ConfigurationScriptBase|Payload"
 msgstr "ConfigurationScriptBase | Carga útil"
 
+msgid "ConfigurationScriptBase|Payload error"
+msgstr "ConfigurationScriptBase | Erro de carga útil"
+
 msgid "ConfigurationScriptBase|Payload type"
 msgstr "ConfigurationScriptBase | Tipo de carga útil"
 
+msgid "ConfigurationScriptBase|Payload valid"
+msgstr "ConfigurationScriptBase | Carga útil válida"
```

Comment:
Same as others, space was after Base in the existing translations, so they did it again.

Reply:
fixed